packages/devtools-reps/src/object-inspector/reducer.js

```diff
@@ -49,6 +50,20 @@ function reducer(
     return cloneState();
   }
 
+  if (type === "GETTER_INVOKED") {
+    return cloneState({
+      actors: data.actor
+        ? new Set(state.actors || []).add(data.result.from)
+        : state.actors,
+      evaluations: new Map(state.evaluations).set(data.node.path, {
+        getterValue:
+          data.result &&
+          data.result.value &&
+          (data.result.value.return || data.result.value.throw)
```

Comment:
what's this doing?

Reply:
assigning the right result to `getterValue`, since the getter might throw.
This can be moved elsewhere in a small util function (or in the action)